src/diary_ui.cpp

```diff
@@ -93,56 +93,75 @@ void print_list_scrollable( catacurses::window *win, const std::string &text, in
     print_list_scrollable( win, list, selection, active, border, color_error );
 }
 
-void draw_diary_border( catacurses::window *win, const nc_color &color = c_white )
+// Print multiple strings, stacked vertically downwards from the specified point
+template<typename ...Ts>
+void mvwprintwa( const catacurses::window &win, point p, Ts... args )
 {
-    wattron( *win, color );
```

Comment:
I intend to bring back the configurable color (even though it wasn't used, I have a use in mind for it) in a future PR that eliminates a lot of unnecessary (and sometimes unwanted or counterproductive) color switching within functions like mvwputch and mvwhline.